Source/WebKit/WebProcess/InjectedBundle/API/c/WKBundleFramePrivate.h

```diff
@@ -57,6 +57,8 @@ WK_EXPORT void WKBundleFrameFocus(WKBundleFrameRef frame);
 
 WK_EXPORT void _WKBundleFrameGenerateTestReport(WKBundleFrameRef, WKStringRef message, WKStringRef group);
 
+WK_EXPORT void* _WKAccessibilityRootObject(WKBundleFrameRef frame);
```

Comment:
It would be nice if we could use the right type instead of void* somehow, but void* is the status quo, so probably not important for this PR.

Also, the underscore is probably not necessary.  It's a convention for ObjC that isn't public API, but this is C and all the bundle isn't public API.